src/main.py

```
@@ -0,0 +1,33 @@
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"status": "ok", "service": "VRAI Simulation Data Analyzer API"}
+
+
+@app.post("/analyze")
+def analyze_data(
+    request: AnalysisRequest, container: ContainerProxy = Depends(get_container)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)